composer.json

```diff
@@ -23,8 +23,9 @@
     "php" : "^7.0",
     "illuminate/contracts": "~5.1.0|~5.2.0|~5.3.0",
     "illuminate/support": "~5.1.0|~5.2.0|~5.3.0",
-    "google/apiclient" : "^1.1",
-    "nesbot/carbon": "^1.21"
+    "google/apiclient" : "^2.0",
+    "nesbot/carbon": "^1.21",
+    "tedivm/stash": "^0.14.1"
```

Comment:
If that bridge is not complex, I'd prefer that over adding an dependency to stash.